pyproject-tensorflow.toml

```diff
@@ -39,6 +39,19 @@ before-all = [
 test-command = "python -c 'import essentia; import essentia.standard; import essentia.streaming; from essentia.standard import MonoLoader, MetadataReader, YamlInput, Chromaprinter, TensorflowPredict'"
 
 
+[[tool.cibuildwheel.overrides]]
+select = "*arm64*"
```

Comment:
Make the override `select` more specific so that it only triggers macos builds (to avoid a conflict with future arm64 Linux builds).